pkg/module/init.go

```diff
@@ -4,16 +4,15 @@ import (
 	"github.com/open-component-model/ocm/pkg/contexts/oci"
 	"github.com/open-component-model/ocm/pkg/contexts/oci/repositories/ocireg"
 	"github.com/open-component-model/ocm/pkg/contexts/ocm"
-	"github.com/open-component-model/ocm/pkg/contexts/ocm/compdesc"
-	"github.com/open-component-model/ocm/pkg/contexts/ocm/compdesc/versions/ocm.software/v3alpha1"
+	// nolint:revive
+	_ "github.com/open-component-model/ocm/pkg/contexts/ocm/compdesc/versions"
```

Comment:
This import registers two schemes: `v3alpha1` and `v2`:
```
import (
        _ "github.com/open-component-model/ocm/pkg/contexts/ocm/compdesc/versions/ocm.software/v3alpha1"
        _ "github.com/open-component-model/ocm/pkg/contexts/ocm/compdesc/versions/v2"
)
```
The existing code registered just `v3alpha1`, in the removed line number 16.
It's the same in the ocm library version `v0.3.0`
Maybe it's safer to do unnamed import of just `v3alpha1`, like this:
```
_ "github.com/open-component-model/ocm/pkg/contexts/ocm/compdesc/versions/ocm.software/v3alpha1"
```
What do you think?